pkg/cluster/cluster_config_test_helpers.go

```diff
@@ -0,0 +1,29 @@
+package cluster
+
+import (
+	"context"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/opendatahub-io/opendatahub-operator/v2/api/common"
+)
+
+// TestClusterConfig holds test configuration for cluster config.
+type TestClusterConfig struct {
+	Platform             common.Platform
+	ApplicationNamespace string
+}
+
+func SetTestClusterConfig(cfg TestClusterConfig) {
+	clusterConfig.Release.Name = cfg.Platform
+	clusterConfig.ApplicationNamespace = cfg.ApplicationNamespace
+}
+
+func ResetTestClusterConfig() {
+	clusterConfig.ApplicationNamespace = ""
+	clusterConfig.Release.Name = ""
+}
+
+func SetApplicationNamespaceForTest(ctx context.Context, cli client.Client) error {
```

Comment:
This functions could be used outside cluster, and also if named as `Test` it could lead to wrong usage. 

What about move the specific `TestGetApplicationNamespace` tests in a test file which points to `package cluster`? In this way, we could avoid to have those functions.

GetApplicationNamespace external API is already tested also in `TestSetApplicationNamespace`

Reply:
updated